tests/mime/mime_tests.cpp

```diff
@@ -21,10 +21,8 @@ static std::string getLargeData() {
   return Base64::Base64Encode(s);
 }
 
-static const std::string kSmallData = Base64::Base64Encode("This is some test data for the file.");
```

Comment:
Building this as a shared library resulted in the following seg fault and I've got no clue what caused it how this change fixed it :thinking: 

```
ninja: Entering directory `/home/avallach/Documents/projects/git/very-simple-smtps/.conan'         
ninja: no work to do.                                                                              
1/1 smtp_lib_tests RUNNING                                                                         
>>> MALLOC_PERTURB_=60 /home/avallach/Documents/projects/git/very-simple-smtps/.conan/tests/smtp_te
sts                                                                                                
―――――――――――――――――――――――――――――――――――――――――――――― ✀  ―――――――――――――――――――――――――――――――――――――――――――――――
AddressSanitizer:DEADLYSIGNAL                                                                      
=================================================================                                  
==9929==ERROR: AddressSanitizer: SEGV on unknown address 0x000000000015 (pc 0x5596ec56344c bp 0x000
000000003 sp 0x7ffc4e02b460 T0)                                                                    
==9929==The signal is caused by a READ memory access.
==9929==Hint: address points to the zero page.   
    #0 0x5596ec56344c in smtp::base64Encode(std::__cxx11::basic_string<char, std::char_traits<char>
, std::allocator<char> > const&, std::vector<unsigned char, std::allocator<unsigned char> > const&)
 (/home/avallach/Documents/projects/git/very-simple-smtps/.conan/tests/smtp_tests+0xb244c)         
    #1 0x5596ec563573 in smtp::Base64::Base64Encode[abi:cxx11](std::vector<unsigned char, std::allo
cator<unsigned char> > const&) (/home/avallach/Documents/projects/git/very-simple-smtps/.conan/test
s/smtp_tests+0xb2573)
    #2 0x5596ec5636bb in smtp::Base64::Base64Encode(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) (/home/avallach/Documents/projects/git/very-simple-smtps/.
conan/tests/smtp_tests+0xb26bb)                                                                    
    #3 0x5596ec5309ae in __static_initialization_and_destruction_0(int, int) [clone .constprop.0] (
/home/avallach/Documents/projects/git/very-simple-smtps/.conan/tests/smtp_tests+0x7f9ae)
    #4 0x5596ec917824 in __libc_csu_init (/home/avallach/Documents/projects/git/very-simple-smtps/.
conan/tests/smtp_tests+0x466824)                                                                   
    #5 0x7fdadca32c99 in __libc_start_main ../csu/libc-start.c:264
    #6 0x5596ec531a29 in _start (/home/avallach/Documents/projects/git/very-simple-smtps/.conan/tes
ts/smtp_tests+0x80a29)                           
                                                 
AddressSanitizer can not provide additional info.                                                  
SUMMARY: AddressSanitizer: SEGV (/home/avallach/Documents/projects/git/very-simple-smtps/.conan/tes
ts/smtp_tests+0xb244c) in smtp::base64Encode(std::__cxx11::basic_string<char, std::char_traits<char
>, std::allocator<char> > const&, std::vector<unsigned char, std::allocator<unsigned char> > const&
)                                                                                                  
==9929==ABORTING                                                                                   
―――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
1/1 smtp_lib_tests FAIL            0.12s   exit status 1                                  
                                                                                                   
                                                 
Summary of Failures:                                                                               
                                                                                                   
1/1 smtp_lib_tests FAIL            0.12s   exit status 1                                         
                                                                                                   
Ok:                 0                                                                              
Expected Fail:      0                                                                              
Fail:               1                                                                              
Unexpected Pass:    0                                                                              
Skipped:            0                                                                              
Timeout:            0                                          
```